src/components/Register/AccountDetails/SaveAccount/SaveAccount.js

```diff
@@ -37,13 +38,24 @@ export default class SaveAccount extends React.Component {
           value={this.props.label}
           onChange={this.handleChangeLabel}
         />
-        <Button
-          className={styles.button}
-          disabled={isEmpty(this.props.label)}
-          onClick={this.handleSave}
-        >
-          Save
-        </Button>
+        <div className={styles.saveButtons}>
+          <Button
+            id={styles.buttonNew}
```

Comment:
We shouldn't use `id` for styling, just `className`.  I think we can style these buttons the way you want without providing distinct names, I'll add some comments to the `scss` file to clarify.